lighthouse-core/config/default.js

```diff
@@ -285,6 +286,7 @@ module.exports = {
         {id: 'uses-long-cache-ttl', weight: 0, group: 'perf-info'},
         {id: 'dom-size', weight: 0, group: 'perf-info'},
         {id: 'critical-request-chains', weight: 0, group: 'perf-info'},
+        {id: 'network-requests', weight: 0, group: 'perf-info'},
```

Comment:
leave off the group name and it'll be excluded from the report, a la `screenshot-thumbnails`?

Reply:
yeah sure sg